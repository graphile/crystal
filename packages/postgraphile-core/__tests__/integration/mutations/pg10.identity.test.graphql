#> schemas: "pg10"

mutation {
  a: createAlwaysAsIdentity(input:{
    alwaysAsIdentity: {
      t: "test"
    }
  }) {
    alwaysAsIdentity {
      id
      t
    }
  }
  b: createByDefaultAsIdentity(input:{
    byDefaultAsIdentity: {
      t: "test"
    }
  }) {
    byDefaultAsIdentity {
      id
      t
    }
  }
  c: createByDefaultAsIdentity(input:{
    byDefaultAsIdentity: {
      id: 100
      t: "test"
    }
  }) {
    byDefaultAsIdentity {
      id
      t
    }
  }
}
