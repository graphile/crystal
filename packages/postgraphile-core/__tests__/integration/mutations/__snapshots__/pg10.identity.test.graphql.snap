// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "a": Object {
      "alwaysAsIdentity": Object {
        "id": 1,
        "t": "test",
      },
    },
    "b": Object {
      "byDefaultAsIdentity": Object {
        "id": 1,
        "t": "test",
      },
    },
    "c": Object {
      "byDefaultAsIdentity": Object {
        "id": 100,
        "t": "test",
      },
    },
  },
}
`;
